packages/sdk/src/lend/providers/morpho/sdk.ts

```diff
@@ -90,6 +91,44 @@ export function calculateBaseApy(vault: any): number {
   }
 }
 
+/**
+ * TEMPORARY - To be removed in https://github.com/ethereum-optimism/verbs/issues/112
```

Comment:
is the plan to eventually just read this information directly onchain?

Reply:
That's the plan, though I wonder if it'll require indexing? 🤔

Reply:
if we wanted to optimize it maybe, but to keep it stateless we can just read directly from the contracts. here is an example: https://github.com/gelatodigital/gelato-smart-wallet-sdk-morpho-demo/blob/3421d85f9e9e3c39a634f59f9bf43dd3dac0543b/app/dashboard/earn/page.tsx#L118-L183